build_bundle_image.sh

```diff
@@ -330,6 +345,15 @@ function prepare_temp_directory {
 
 	local latest_tomcat_version=$(get_latest_tomcat_version "${tomcat_version}")
 
+	if [ -z "${latest_tomcat_version}" ]
+	then
+		lc_log ERROR "Unable to retrieve the latest Tomcat version."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
```

Comment:
@kiwm, why don't you use a `exit` here instead? This way, we would save the need to declare the new lines from `261` to `265`. Actually, why not do it on the function `get_latest_tomcat_version`?